backend/src/main/kotlin/app/ehrenamtskarte/backend/cards/webservice/schema/CardMutationService.kt

```diff
@@ -147,6 +147,9 @@ class CardMutationService {
 
         val userEntitlements = transaction { UserEntitlementsRepository.findUserEntitlements(userHash.toByteArray()) }
         if (userEntitlements == null || userEntitlements.revoked || userEntitlements.endDate.isBefore(LocalDate.now())) {
+            if (userEntitlements == null) {
+                logger.info("${context.remoteIp} failed to create a new card")
```

Comment:
:thought_balloon: any chance that, without knowing the context, someone could remove or edit this?
Maybe a comment is needed explaining why this was added, since it's not clear from the code itself?

Reply:
yes your are right. its not a particular issue in this pr but we should in general add a comment to all ban related logs. But could be done in a different task but please create one (someone :))

Reply:
Done for all found occurrences.